core/flyout_horizontal.js

```diff
@@ -289,7 +291,7 @@ Blockly.HorizontalFlyout.prototype.layout_ = function(contents, gaps) {
       var blockHW = block.getHeightWidth();
 
       // Figure out where to place the block.
-      var tab = block.outputConnection ? Blockly.BlockSvg.TAB_WIDTH : 0;
+      var tab = block.outputConnection ? this.tabWidth_ : 0;
```

Comment:
I think this is backwards. 
In RTL the margin on the left side is bigger for blocks with outputs compared to non-outputs
![image](https://user-images.githubusercontent.com/11066279/64740410-1e561e80-d4aa-11e9-8038-92a981fd1e2d.png)
![image](https://user-images.githubusercontent.com/11066279/64740426-30d05800-d4aa-11e9-8fee-0fd4e06b59e7.png)


In both cases, we're missing margin on the right side when in rtl:
![image](https://user-images.githubusercontent.com/11066279/64740372-01215000-d4aa-11e9-8ead-1525a9e03e6f.png)

Reply:
Sorry, first two images are LTR. The right side margin is missing in both RTL and LTR.

Reply:
I think you are right. I'll fix it since it's likely the intended behavior.